torch_geometric/transforms/compose.py

```diff
@@ -8,7 +8,7 @@ class Compose(BaseTransform):
     """Composes several transforms together.
 
     Args:
-        transforms (List[Callable]): List of transforms to compose.
+        transforms ([callable]): List of transforms to compose.
```

Comment:
I feel `List[Callable]` is better than `[callable]` since `[]` may sometime mean optional.

Reply:
Resolved.